pkg/console/install_panels.go

```diff
@@ -291,6 +276,14 @@ func addDiskPanel(c *Console) error {
 				return updateValidatorMessage(err.Error())
 			}
 
+			if err := validateDiskSizeSoft(device); err != nil && !userInputData.HasWarnedDiskSize {
+				userInputData.HasWarnedDiskSize = true
+				return updateValidatorMessage(fmt.Sprintf(
+					"Warning: %s.\nThis might cause system to become unstable. Press Enter continue.",
```

Comment:
probably showing the `Disk size smaller than recommended minimum size 140G. Presse Enter to continue.` is sufficient enough.

Reply:
Done